srcalc/src/main/webapp/WEB-INF/views/tiles/enterVariables.jsp

```diff
@@ -99,17 +105,17 @@
         <script type="text/javascript" src="${dataTablesUrl}"></script>
         <script type="text/javascript">
         $(document).ready(function(){
-        	initEnterVariablesPage();
         	// Set up the properties for the procedures DataTable
-	    	var table = $("#procedureTable").dataTable( {
-	    		// Make the radio button column smaller so that IE
-	    		// will adjust column widths properly.
+	    	$("#procedureTable").dataTable({
+	    		"data": procedureArray,
 	    		"aoColumnDefs":[
+				//Make the radio button column smaller so that IE
+				// will adjust column widths properly.
 	    		{
 	    			"width": "10%",
 	    			"targets": [0]
 	    		},
-	    		// Make the select column,  and RVU unsearchable
+	    		// Make the select column, description,  and RVU unsearchable
 	    		{
 	    			"bSearchable": false, 
 	    			"aTargets": [0,2,3]
```

Comment:
Since there are only 4 columns, it may be clearer to just use the "columns" option instead of multiple columnDefs.